BrainPortal/lib/cbrain_task_generators/schema_task_generator.rb

```diff
@@ -232,6 +232,11 @@ def register(task,create_tool_config)
       # sense on the portal).
       return if Rails.root.to_s =~ /BrainPortal$/
 
+      # Create a ToolConfig iff 
+      #   (1) the Bourreau has a docker executable and 
+      #   (2) the descriptor specifies a docker image
+      return if docker_image.nil? || resource.docker_executable_name.nil?
```

Comment:
Be careful here: some Bourreaux may have Docker installed but no Docker executable defined. The Docker executable defaults to "docker", it's only that some clusters have wrappers for it (e.g. 'udocker').